src/Controller/TaskController.php

```diff
@@ -121,7 +121,7 @@ public function taskListAction($page)
         ], false);
 
          //Check the user timezone
-         if ($timezone == '') {
+        if (empty($timezone)) {
```

Comment:
@CamillePMyddleware , the variable $timezone isn't initialized here, maybe a code lost in a merge. Could you check this ?